README.adoc

```diff
@@ -97,6 +97,20 @@ embedded.kafka.topicsToCreate=some_topic
 - https://dev.to/sivalabs/springboot-integration-testing-using-testcontainers-starter-13h2[SpringBoot Integration Testing using TestContainers Starter]
 - https://alexromanov.github.io/2019/04/02/spring-boot-docker-containers/[Easy way to test Spring Boot microservices]
 
+== Common configuration options
+
+./src/test/resources/application.properties
+[source,properties]
+----
+embedded.containers.forceShutdown=true //default is false
+----
+
+./src/test/resources/bootstrap.properties
+[source,properties]
+----
+embedded.containers.enabled=true //default is true
```

Comment:
please document the priority of the properties.

Reply:
done